projects/reservoir-protocol/index.js

```diff
@@ -1,22 +1,31 @@
 const config = {
-  ethereum: [
-    // https://docs.reservoir.xyz/products/proof-of-reserves
-    '0x0c7e4342534e6e8783311dCF17828a2aa0951CC7',
-    '0x9BB2c38F57883E5285b7c296c66B9eEA4769eF80',
-    '0x99A95a9E38e927486fC878f41Ff8b118Eb632b10',
-    // '0x31Eae643b679A84b37E3d0B4Bd4f5dA90fB04a61', - exluded RUSD because it is project's own token
-  ]
+    ethereum: [
+        // https://docs.reservoir.xyz/products/proof-of-reserves
+        '0x0c7e4342534e6e8783311dCF17828a2aa0951CC7',
+        '0x9BB2c38F57883E5285b7c296c66B9eEA4769eF80',
+        '0x99A95a9E38e927486fC878f41Ff8b118Eb632b10',
+        '0x31Eae643b679A84b37E3d0B4Bd4f5dA90fB04a61'
```

Comment:
why do you want to include this here? backing of rUSD is the other vaults right?

Reply:
It's all the assets of the protocol, back our liabilities, including rUSD.

Reply:
so, counting rUSD as tvl here would be doublecounting?

Reply:
Well no, because we add the other side, liability, on as "borrowed".

Reply:
sorry, what is backing the rUSD, the assets that we are counting as tvl right?

Reply:
The full balance sheet of the protocol, i.e. for our liabilities, all the assets. The RWA loans we issue, our lending are assets within Reservoir.

Reply:
It's that our debt is offset by assets, so we want to have a way to show our liabilities in that the value is offset by the total assets held in by the protocol.

Reply:
We want to add all the additional asset we have, so will offset by the liabilities we have.

Reply:
@eukadish if the assets are not locked in your contracts, liabilities or not, we can track them only under borrowed.

Reply:
the reason we are stubborn about this is, we have had issues where protocols were inflating tvl by cyclic lending

Reply:
Understood, understood. Okay, then let me sync with my team and see if we can break it down better or maybe change how we represent our numbers and get back to you.

Reply:
hi @eukadish I am closing this for now, we can reopen the PR later